src/Microsoft.Diagnostics.DebugServices/IConsoleService.cs

```diff
@@ -29,6 +29,12 @@ public interface IConsoleService
         /// <param name="value"></param>
         void WriteError(string value);
 
+        /// <summary>Writes Debugger Markup Language (DML) markup text.</summary>
+        void WriteDml(string text);
```

Comment:
This is fine for now. I was thinking about a more abstract "IFormattingService" with "formatter" classes that DML could be abstracted behind and maybe work for other platforms/UIs like VS.  Something for the future.